test/runtests.jl

```diff
@@ -68,7 +61,7 @@ end
 """
     experimental_order_of_convergence(errors, dts)
 
-Compute the experimental order of convergence for given `errors` and
+Compute the experimental orders of convergence for given `errors` and
 time step sizes `dts`.
 """
 function experimental_order_of_convergence(errors, dts)
```

Comment:
```suggestion
function experimental_orders_of_convergence(errors, dts)
```

Since it's now plural, this should be reflected in the function name (also where it is used)